scripts/aggregate-translations/aggregate-translations.js

```diff
@@ -19,22 +19,19 @@ const customDirectories = (baseDirectory, directories) =>
 
 const isFile = filePath => (fse.existsSync(filePath) && !fse.lstatSync(filePath).isDirectory());
 
-const configFile = (configPath) => {
-  if (configPath) {
+const loadConfigFile = (configPath) => {
+  const i18nConfigPath = path.resolve(process.cwd(), configPath);
+  if (isFile(i18nConfigPath)) {
```

Comment:
If we default the config pattern to be `./terrai18n.config`, then we wouldn't want would want a hard fail if a config file isn't there, it would just return `{}`.

Above you mention we should default the configPath to undefined, however in the current configuration, if this value is not defined, we then search for `./terrai18n.config` before returning `{}`, so this is making this default config location visible.

I do see value in quickly failing on passed config paths when provided.

Reply:
I'm fine with leaving how the current implementation is.